joy.py

```diff
@@ -249,8 +249,7 @@ def render(self):
         svg_header = render_tag(**attrs)+ "\n"
         svg_footer = "</svg>\n"
 
-        # flip the y axis so that y grows upwards
-        node = Group(self.nodes) | Scale(sx=1, sy=-1)
+        node = Group(self.nodes)
```

Comment:
I don't think replacing `y` to `-y` wherever it is used for avoiding this transformation is a good idea. Right now we don't maintain the exact attributes used to create a shape, but we would like to do that at some point.

I think it would better to apply `scale(y=-1)` to text shape to compesate for this.